src/test/java/com/google/sps/MatchRankingTest.java

```diff
@@ -331,4 +366,33 @@ private void testOnlyAddNewBio(String newBio, User user) {
     user.setBioMap(savedMatchWordCount);
   }
 
+  private String getRandomGoodBio(int bioLength) {
+      String[] possibleWords = {"security", "engineer", "software", "Security", "data", "work", "secure", "engineering", "work", "google", "microsoft", "structures",
+      "working", "network", "securing", "endpoints", "cloud", "android", "app", "application"};
+
+      Random random = new Random();
+      String toReturn = "";
```

Comment:
Consider using a StringBuilder which should be more efficient when appending strings in a loop.

Reply:
Done